base/applications/games/spider/lang/ro-RO.rc

```diff
@@ -27,20 +27,20 @@ BEGIN
     CONTROL "", IDC_CARDBACK10, "Static", SS_NOTIFY, 175, 89, 50, 71
     CONTROL "", IDC_CARDBACK11, "Static", SS_NOTIFY, 232, 89, 50, 71
     CONTROL "", IDC_CARDBACK12, "Static", SS_NOTIFY, 289, 89, 50, 71
-    DEFPUSHBUTTON "Con&firmă", IDOK, 114, 165, 54, 14
-    PUSHBUTTON "A&nulează", IDCANCEL, 176, 165, 54, 14
+    DEFPUSHBUTTON "OK", IDOK, 114, 165, 54, 14
+    PUSHBUTTON "Revocare", IDCANCEL, 176, 165, 54, 14
 END
 
 IDD_DIFFICULTY DIALOGEX 100, 100, 106, 80
 STYLE DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU | DS_SHELLFONT
-CAPTION "Joc la categoria"
+CAPTION "Dificil"
```

Comment:
Dificil? Perhaps using "selectare dificultate" or something similar fits better.

Reply:
It's fixed now. Now there is Selectare nivel de dificultate.